### PR TITLE
Single level usage as built 2.0.0

### LINKED ISSUE #196: [Model Update]: SingleLevelUsageAsBuilt: prepare for Decentralized Digital Twin Registry
## Update Reason
In order to support efficient data chain building in a environment with a decentralized digital twin registry, the identifier of a business Partner who owns a referenced Digital Twin must be included in the reference. 

## Sample Data

See updated model: 
#198 

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists
- [x] The potential updates are discussed with all stakeholders
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [x] The potential update will be either 
  - [ ] backward compatible or 
  - [x] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed

### PR DESCRIPTION
**Pull request by jacewski-bosch**
## Description

In order to support efficient data chain building in a environment with a decentralized digital twin registry, the identifier of a business Partner who owns a referenced Digital Twin must be included in the reference.

Closes #196 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  BAMM CLI is available [here](https://openmanufacturingplatform.github.io/sds-documentation/sds-developer-guide/dev-snapshot/tooling-guide/bamm-cli.html) and in [GitHub](https://github.com/OpenManufacturingPlatform/sds-sdk/releases)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "name" and "description"** in English language. 
- [x] **no duplicate names or preferredNames** within an Aspect (e.g. a Property and the referenced Characteristic should not have the same name)
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the BAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [ ] All required reviewers have approved this PR (see reviewers section)
- [ ] The new aspect (version) will be implemented by at least one data provider
- [ ] The new aspect (version) will be consumed by at least one data consumer
- [ ] There exists valid test data
- [ ] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [ ] The model has at least version '1.0.0'

### CODE REVIEW

File: io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl [hunk 1]
```diff
@@ -0,0 +1,155 @@
+#######################################################################
```
Comment (bs-jokri) [inline io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl:1]: metadata.json missin

File: io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:1.0.0#> .
```
Comment (bs-jokri) [inline io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl:26]: switch to bAMM 2.0.0 or if model can be released in 2 weeks, than switch to sAMM

File: io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+   bamm:description "The aspect provides the information in which parent part(s)/product(s) the given item is assembled in. Could be a 1:1 relationship in terms of a e.g. a brake component or 1:n for e.g. coatings. The given item as well as the parent item must refer to an object from as-built lifecycle phase, i.e. a batch or a serialized part. "@en ;
+   bamm:properties ( :catenaXId :customers ) ;
+   bamm:operations ( ) ;
+   bamm:events ( ) .
+
+:catenaXId a bamm:Property ;
+   bamm:preferredName "Catena-X Identifier"@en ;
+   bamm:description "The Catena-X ID of the given part (e.g. the assembly), valid for the Catena-X dataspace."@en ;
+   bamm:characteristic :CatenaXIdTraitCharacteristic ;
+   bamm:exampleValue "urn:uuid:055c1128-0375-47c8-98de-7cf802c3241d" .
```
Comment (bs-jokri) [inline io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl:46]: is this validating against regex (ie json example payload validates against generated json schema)

**Review by bs-jokri — CHANGES_REQUESTED** [review]
add metadata and check regex

**Comment by bs-jokri** [comment]
model does not validate:
```
java -jar ~/Projects/BAMMmodels/.BAMMCLI/bamm-cli-2.1.3.jar  aspect io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl validate
Validation failed:
Resolution strategy returned a model which does not contain element definition for urn:bamm:io.catenax.single_level_usage_as_built:2.0.0#SingleLevelUsageAsBuilt

```

File: io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl (hunk 2 above)
Comment (jacewski-bosch) [reply io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl]: switched to samm

**Comment by jacewski-bosch** [comment]
> ### Validation Report for io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl
> #### Validation failed:

 This seems to be due to BAMM to SAMM change. I guess this is a general issue with the github-action validation.

**Comment by jacewski-bosch** [comment]
> model does not validate:
> 
> ```
> java -jar ~/Projects/BAMMmodels/.BAMMCLI/bamm-cli-2.1.3.jar  aspect io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl validate
> Validation failed:
> Resolution strategy returned a model which does not contain element definition for urn:bamm:io.catenax.single_level_usage_as_built:2.0.0#SingleLevelUsageAsBuilt
> ```

the json schema and json example as created by AME 4.4.1 (based on ESMF SDK 2.2.0) validate.

File: io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl (hunk 3 above)
Comment (jacewski-bosch) [reply io.catenax.single_level_usage_as_built/2.0.0/SingleLevelUsageAsBuilt.ttl]: the json example as created by AME 4.4.1 (based on ESMF SDK 2.2.0) validate against the json schema created the same way.

**Comment by jonbckr** [comment]
modeling_team

**Comment by bs-jokri** [comment]
checked json schema validation: OK

**Comment by bs-jokri** [comment]
superseded by https://github.com/eclipse-tractusx/sldt-semantic-models/pull/287

**Pull request closed without merging**